packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -316,71 +70,6 @@ QUnit.test('properties within an if statement should not fail on re-render', fun
   equal(view.$().text(), '');
 });
 
-QUnit.test('should update the block when object passed to #if helper changes', function() {
-  owner.register('template:menu', compile('<h1>{{#if view.inception}}{{view.INCEPTION}}{{/if}}</h1>'));
-
-  view = EmberView.create({
-    [OWNER]: owner,
-    templateName: 'menu',
-    INCEPTION: 'BOOOOOOOONG doodoodoodoodooodoodoodoo',
-    inception: 'OOOOoooooOOOOOOooooooo'
-  });
-
-  runAppend(view);
-
-  equal(view.$('h1').text(), 'BOOOOOOOONG doodoodoodoodooodoodoodoo', 'renders block if a string');
-
-  var tests = [false, null, undefined, [], '', 0];
-
-  tests.forEach(function(val) {
-    run(function() {
-      set(view, 'inception', val);
-    });
-
-    equal(view.$('h1').text(), '', `hides block when conditional is "${val}"`);
-
-    run(function() {
-      set(view, 'inception', true);
-    });
-
-    equal(view.$('h1').text(), 'BOOOOOOOONG doodoodoodoodooodoodoodoo', 'precond - renders block when conditional is true');
-  });
-});
-
-QUnit.test('should update the block when object passed to #if helper changes and an inverse is supplied', function() {
```

Comment:
Removed without replacement (?). Seems like this is about the content of the bock get updated? I can add it back if it is important. (This compared to the above is to test with or without inverse. Do we need both in the new tests?)